src/styling/ProjectWorkPage.css

```diff
@@ -76,17 +67,73 @@
   margin-bottom: 4px;
 }
 
-.project-org-name {
-  margin-bottom: 48px;
+.title-container {
+  margin-bottom: 3rem;
+}
+
+.title-container p {
+  font-size: 24px;
+  margin-bottom: 0;
+}
+
+.info-container {
+  margin-bottom: 2rem;
 }
 
 .project-detail-email {
   white-space: pre-line;
-  line-height: 36px;
+  line-height: 1.3rem;
+  font-size: 1rem;
+  margin: 2em 0;
+  padding: 2em;
+  background-color: var(--light-yellow);
+  color: var(--main-grey);
 }
 
-@media only screen and (max-width: 768px) {
-  .collapsable-content {
-    padding: 24px;
+@media only screen and (max-width: 992px) {
+  .project-detail-email {
+    padding: 1.1rem;
+  }
+}
+
+.project-detail-email .buttons-container {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.project-detail-email .buttons-container .btn {
+  font-size: 16px;
+  max-width: 300px;
+  margin: 0 0.3rem;
+  border-radius: 5px;
+  padding: 0.8rem;
+}
+
+.project-detail-email .primary-button.clicked {
+  background-color: var(--dark-yellow);
+  border-color: var(--dark-yellow);
+}
+
+@media only screen and (max-width: 992px) {
```

Comment:
can we group all mobile styles together? ie putting this and the other <992 style above all together instead of grouped with their full size counterparts so that it's easier to keep track of what all the differences are between screen sizes